pkg/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -1473,6 +1473,64 @@ func TestTombstoneUpdate(t *testing.T) {
 	dst.CheckQueryResults(t, "SELECT * FROM tab WHERE pk = 1", [][]string{})
 }
 
+func TestReplicateComputedColumns(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	skip.UnderDeadlock(t)
+	defer log.Scope(t).Close(t)
+
+	ctx := context.Background()
+
+	server, s, dbSource, dbDest := setupLogicalTestServer(t, ctx, testClusterBaseClusterArgs, 1)
+	defer server.Stopper().Stop(ctx)
+
+	// Note: the primary key is basicaly (a, b), but with a computed column that
+	// is d=a+b.
+	createTableStmt := `CREATE TABLE computed_cols (
```

Comment:
nit: if you dont mind, could you rename the table used in TestOptimsitcInsertCorruption :D

Reply:
Fixed